static/js/utils/google_signin.tsx

```diff
@@ -22,32 +22,27 @@ import {
   User,
   UserCredential,
 } from "firebase/auth";
-import React from "react";
 
 import { auth } from "./firebase";
 
-interface GoogleSignInProps {
-  scopes: string[];
-  onSignIn: (user: User, credential: OAuthCredential) => void;
-}
-
-export function GoogleSignIn(props: GoogleSignInProps): JSX.Element {
-  const signInWithGoogle = (): void => {
-    const provider = new GoogleAuthProvider();
-    for (const scope of props.scopes) {
-      provider.addScope(scope);
-    }
-    signInWithPopup(auth, provider)
-      .then((result: UserCredential) => {
-        props.onSignIn(
-          result.user,
-          GoogleAuthProvider.credentialFromResult(result)
-        );
-      })
-      .catch((error: AuthError) => {
-        // Handle errors here
-        console.error(error);
-      });
-  };
-  return <button onClick={signInWithGoogle}>Sign In with Google</button>;
+export function signInWithGoogle(
```

Comment:
nice! and just to confirm, will this work if the user isn't signed into their google account on the browser? e.g., using safari or incognito

Reply:
yes, in that case the pop-up [won't auto-dismiss](https://github.com/datacommonsorg/website/assets/10565461/82a84641-eaa6-42e9-88a2-136872ee55e8).